### ISSUE TITLE
JSON linter thinks a list of JSON objects are incorrect

### ISSUE THREAD

**Issue by CGBassPlayer** [riok/Kreya#102]
**Describe the bug**
This is a very low priority bug. I would not be surprised if this issue stems from an external library instead of your codebase. But the linter in the JSON body section does not think a list of JSON objects are valid. 

Obviously this does not stop my workflow, its more annoying than anything else.

**To Reproduce**
1. Create REST call
2. Add JSON body
3. Put a list of objects in the body
```json
[
  {
    "code": "TST-BUAD",
    "name": "Business Administration",
    "customCode": "TST",
    "isActive": true
  },
  {
    "code": "TST-ENGL",
    "name": "English",
    "customCode": "TST",
    "isActive": true
  }
]
```

**Expected behavior**
List of JSON objects being valid and no warning lines under the entire body

**Screenshots**
Here is a screenshot showing the warning
![image](https://user-images.githubusercontent.com/34293941/186767906-95345b05-3c4b-41d0-8d89-21b0b421a390.png)

**Environment**
```json
{
  "kreyaVersion": "1.8.1",
  "platform": "Win32",
  "userAgent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/104.0.5112.102 Safari/537.36 Edg/104.0.1293.63"
}
```

**Comment by CommonGuy** [comment]
Thanks for the bug report. This is a bug on our side, since we define that the request should be an object, even though it could be a list (or other types). We will fix this in the next version.

**Comment by CommonGuy** [comment]
We just released Kreya 1.9, which fixes this behaviour.
